webpack/webpack.config.js

```diff
@@ -1,63 +1,26 @@
 import _ from 'lodash';
-import webpack from 'webpack';
-import strategies from './strategies';
-import yargs from 'yargs';
-
-const argv = yargs
-  .alias('p', 'optimize-minimize')
-  .alias('d', 'debug')
-  .argv;
-
-const defaultOptions = {
-  development: argv.debug,
-  docs: false,
-  test: false,
-  optimize: argv.optimizeMinimize
-};
-
-export default (options) => {
-  options = _.merge({}, defaultOptions, options);
-  const environment = options.optimize ? 'production' : 'development';
-
-  const config = {
-    entry: {
-      'react-bootstrap': './src/index.js'
-    },
-
-    output: {
-      path: './dist',
-      filename: '[name].js',
-      library: 'ReactBootstrap',
-      libraryTarget: 'umd'
-    },
-
-    externals: [
-      {
-        'react': {
-          root: 'React',
-          commonjs2: 'react',
-          commonjs: 'react',
-          amd: 'react'
-        }
+import baseConfig, { argv } from './base.config';
+
+export default _.extend({}, baseConfig, {
+  entry: {
+    'react-bootstrap': './src/index.js'
+  },
+
+  output: {
+    path: './dist',
+    filename: argv.optimizeMinimize ? '[name].min.js' : '[name].js',
+    library: 'ReactBootstrap',
+    libraryTarget: 'umd'
+  },
+
+  externals: [
+    {
+      'react': {
+        root: 'React',
+        commonjs2: 'react',
+        commonjs: 'react',
+        amd: 'react'
```

Comment:
I believe this topic is for another PR :smiley: 
I can create `todo` issue for it.
What do you think ?